eviatar/enable_windows_features.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Install-WindowsFeatures {

```

Comment:
I believe I saw this function in your other script already... ;)

Reply:
The global question and answer regarding the code structure also giving answer to this conversation.